regression-test-all.sh

```diff
@@ -80,7 +80,8 @@ while [ $# -gt -0 ]; do
 done
 [ -x "$EXE" -o "$MAKE" = true ] || die "Executable '$EXE' must exist or you must specify -make"
 
-CORES=${CORES:=1} # cores =1 for now since I broke threading. :-(
+NUM_THREADS=$(getconf _NPROCESSORS_ONLN)
+CORES=$(( NUM_THREADS < 4 ? NUM_THREADS : 4 ))
```

Comment:
I actually meant AT MOST 4, not at least. :-)

Reply:
Fixed, oops!